pkgs/development/tools/mold/default.nix

```diff
@@ -30,6 +31,35 @@ stdenv.mkDerivation rec {
   LTO = 1;
   makeFlags = [ "PREFIX=${placeholder "out"}" ];
 
+  suffixSalt = lib.replaceStrings ["-" "."] ["_" "_"] stdenv.targetPlatform.config;
+  wrapperName = "MOLD_WRAPPER";
+  coreutils_bin = lib.getBin coreutils;
```

Comment:
Where are those three used?

Reply:
They are referenced in templates that get instantiated with the calls to `substituteAll` below. We could just `export` them right in the postInstall script, but defining them in nix code reduces volatility.

i'll add a comment.